app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Api::V1::ReviewsController < ApplicationController
+
+  def index
+    all_reviews = Review.all
+    render json: all_reviews
+  end
+
+
+  def show
+    render json: Review.find(params[:id])
+  end
```

Comment:
This is a BIT nitpicky, but I'd stick to one format here: either set your data as a variable in both index & show, or neither. In other words, either change to `render json: Review.all` or 
```
def show
  review = Review.find(params[:id])
  render json: review
end
```